src/main/java/org/shanerx/tradeshop/utils/objects/ObjectHolder.java

```diff
@@ -115,6 +121,21 @@ public Map<String, Object> asMap() {
         return null;
     }
 
+    public boolean isList() {
+        return !isNull() && obj instanceof List;
+    }
+
+    public Optional<List<String>> asStringList() {
+        Optional<List<String>> ret = Optional.empty();
+        if (isList()) {
+            try {
+                ret = Optional.of((List<String>) obj);
+            } catch (ClassCastException ignored) {
+            }
+        }
+        return ret;
+    }
+
```

Comment:
refer to the above

Reply:
Changed to
`ret = Optional.of(((List<Object> obj).stream().map(Object::toString).collect(java.util.stream.Collectors.toList()));`